SETUP.md

```diff
@@ -71,69 +71,64 @@ Finally, flick the repository's switch to enable CI.
 Generate a deploy key so Travis CI can write to the repository.
 
 ```sh
+# IMPORTANT: change working directory to /ci
 cd ci
+
+# Generate deploy.key.pub (public) and deploy.key (private)
 ssh-keygen \
-  -t rsa \
-  -b 4096 \
-  -C "travis@travis-ci.com" \
-  -N "" \
+  -t rsa -b 4096 -N "" \
+  -C "deploy@travis-ci.com" \
```

Comment:
I like one argument per line in general, but users get overwhelmed by the length of setup, so this is an easy target for condensing it. Especially since the short names are readable anyways.

Reply:
Email address updated based on default used by https://docs.travis-ci.com/user/deployment/pages/